tests/Microsoft.Bot.Builder.Ai.LUIS.Tests/LuisRecognizerTests.cs

```diff
@@ -335,28 +333,19 @@ private JObject JsonLuisResult(RecognizerResult result)
         [TestMethod]
         public async Task Composite1()
```

Comment:
If you don't mind some feedback, these tests are all too complex. Consider having a test that serializes a single intent, a single entity, and so on down the line. These are rich, full-featured, integration tests. I think actual Unit Tests would might be more important. 

... although you may already have all of those on the generation side and I've just not seen them as that code doesn't live in this repo.

Reply:
That is a matter of perspective.  Because this generates what was actually generated, you can run windiff and see the differences.  If you make a systematic change every file would need to be updated.